docs/deployment/connect-scm.md

```diff
@@ -63,52 +63,153 @@ You have successfully connected an org in Semgrep Cloud Platform with an organiz
 
 For users of GitLab cloud-hosted plans, a connection to GitLab is created automatically after [adding a Semgrep job to GitLab CI/CD](/deployment/add-semgrep-to-ci/). No other steps are needed.
 
-## Connect to on-premise GitHub or GitLab orgs
-
-### Applicable plans
-
-| GitHub | GitLab |
-| ------ | ------ |
-| GitHub Enterprise Server       | Any GitLab Self Managed plan        |
-
-### Table of required scopes for PATs
-
-Semgrep Cloud Platform requires personal access tokens (PATs) with assigned scopes. These scopes grant necessary permissions to the PAT and vary depending on the user's plan.
-
-| GitHub Enterprise Server          | GitLab Self-Managed        |
-|:---------------------------|:---------------------------|
-| <ul><li>`public_repo`</li> <li>`repo:status`</li> <li>`user:email`</li> <li>`write:discussion`</li></ul> | `api` |
-
-### Make the connection
+## Connect to on-premise orgs
+
+### GitHub Enterprise Server
+
+This section is applicable to users with **GitHub Enterprise Server**. We
+recommend connecting to your GitHub orgs using the Semgrep App instead of using
+a personal access token (PAT) whenever possible.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+    defaultValue="app"
+    values={[
+    {label: 'Semgrep App (Recommended)', value: 'app'},
+    {label: 'PAT', value: 'pat'},
+    ]}
+>
+
+<TabItem value='app'>
+
+The Semgrep App for GitHub Enterprise (GHE) creates a connection between Semgrep
+and orgs in your GHE deployment. There are two primary installation
+steps:
+
+1. Install the Semgrep App for the first time using the GHE organization (org)
+   that "owns" the app.
+2. Install the app for additional GHE orgs.
+
+<!-->Using HTML headers instead of MD headers because I don't want them in the right-side nav<-->
+<h4>Install the Semgrep App for the first time using the organization that "owns" the app</h4>
+
+When you install the Semgrep App for GHE for the first time in your GHE
+instance, you must choose the organization within your deployment to act as the
+owner of the app. As the owner, this GHE org controls the settings and
+permissions granted to the app.
+
+1. Log in to [Semgrep Cloud Platform](https://semgrep.dev/login/).
+2. Go to **Settings** > **Source Code Managers**, and click **Add GitHub
+   Enterprise**.
+   ![Semgrep Cloud Platform's Source code managers page](/img/ghe-1.png#md-width)
+3. In the popup window, provide:
+   - The **Name of your GitHub Organization**
+   - The **URL** to access your deployment
+   - A random string in the **Access token** field; this field will be optional in the future
```

Comment:
```suggestion
   - A random string in the **Access token** field such as `asdf`;  this field will be optional in the future
```
This has to be the weirdest instruction I've ever seen. It might throw users off so I provided a random value.